data_structures/Heap/Binary_Heap/min_heap_two.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:test/test.dart';
+
+class MinHeap {
+  List<int> heap;
+
+  void buildHeap(List<int> array) {
+    this.heap = _heapify(array);
+  }
+
+  List<int> _heapify(List<int> array) {
+    int firstParent;
+    firstParent = (array.length - 2) ~/ 2;
```

Comment:
You could initialize `firstParent` with this value
```suggestion
    int firstParent = (array.length - 2) ~/ 2;
```